cpanfile

```diff
@@ -3,7 +3,7 @@ requires 'perl' => '5.18.2';
 # Mandatory modules
 requires 'Algorithm::Diff'                            => '1.1902';
 requires 'Authen::Passphrase'                         => '0.008';
-requires 'Captcha::reCAPTCHA'                         => '0.97';
+requires 'Captcha::reCAPTCHA'                         => '0.99';
```

Comment:
ReCAPTCHA 2 isn’t supported before 0.98; as 0.99 fixes some bugs and is in the cpan snapshot file already anyway, why not use that version directly.